enhancements/ingress/default-apps-domain.md

```diff
@@ -0,0 +1,118 @@
+---
+title: default-apps-domain
+authors:
+  - "@dustman9000"
+reviewers:
+  - "@Miciah"
+  - "@danehans"
+  - "@frobware"
+  - "@knobunc"
+  - "@sgreene570"
+approvers:
+  - "@knobunc"
+  - "@Miciah"
+creation-date: 2020-08-12
+last-updated: 2020-09-04
+status: implementable
+---
+
+# Default Apps Domain
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement allows for the configuration of a default apps domain in which
+can be used to set the default host and domain of the route, if not specified by
+the admin. Instead of using the default cluster domain, this new setting can be
+used to override it when set.
+
+## Motivation
+
+Many customers have requested "Custom Domains" in which they would like an already
+installed Openshift cluster to support additional domains for their apps by default.
+This domain is owned by a customer in which they would point a wildcard CNAME record
+to an existing well known CNAME record.
+
+### Goals
+
+1. Enable the cluster administor to specify the default apps domain seperate
+from the installed cluster domain.
+
+## Proposal
+
+The Ingress API is extended by adding an optional `AppsDomain` field with type
+`string`.
+
+```go
+type IngressSpec struct {
+    ...
+    // optional domain to be used instead of Domain installed cluster domain
+    // when Route Subdomain is unspecified.
```

Comment:
I don't understand the phrase "Domain installed cluster domain"; do you mean the value set at install time?
```suggestion
    // appsDomain is an optional domain to use instead of the one specified
    // in the domain field when a Route is created without specifying an explicit
    // host.  If appsDomain is nonempty, this value is used to generate default
    // host values for Route.  Unlike domain, appsDomain may be modified after
    // installation.
```